src/gfx/mesh.hpp

```diff
@@ -0,0 +1,88 @@
+
+#pragma once
```

Comment:
I think we should convert to traditional header guards and stick with them for consistency

Reply:
Changed them